spec/bolt/node/winrm_spec.rb

```diff
@@ -203,19 +203,50 @@ def stub_winrm_to_raise(klass, message)
   end
 
   it "can run a task remotely", winrm: true do
-    contents = 'Write-Output "$env:PT_message_one" ${env:PT_message two}'
+    contents = 'Write-Host "$env:PT_message_one ${env:PT_message two}"'
     arguments = { :message_one => 'task is running',
                   :"message two" => 'task has run' }
     with_tempfile_containing('task-test-winrm', contents) do |file|
       expect(winrm._run_task(file.path, 'environment', arguments).value)
-        .to eq("task is running\r\ntask has run\r\n")
+        .to eq("task is running task has run\r\n")
+    end
+  end
+
+  it "will collect on stdout using valid output mechanisms", winrm: true do
+    contents = <<-PS
+    # explicit Format-Table for PS5+ overrides implicit Format-Table which
+    # includes a 300ms delay waiting for more pipeline output
+    # https://github.com/PowerShell/PowerShell/issues/4594
+    Write-Output "message 1" | Format-Table
+
+    Write-Host "message 2"
+    "message 3" | Out-Host
+    $Host.UI.WriteLine("message 4")
+    [Console]::WriteLine("message 5")
+
+    # preference variable must be set to show Information messages
+    $InformationPreference = 'Continue'
+    Write-Information "message 6"
+    PS
+
+    with_tempfile_containing('stdout-test-winrm', contents) do |file|
+      expect(
+        winrm._run_script(file.path, []).value
+      ).to eq([
+        "message 1\r\n",
+        "message 2\r\n",
+        "message 3\r\n",
+        "message 4\r\n",
+        "message 5\r\n",
+        "message 6\r\n"
+      ].join(''))
```

Comment:
There are other cmdlets for writing output to different streams, but I think we can skip testing those for now - 

* Write-Debug
* Write-Progress
* Write-Verbose
* Write-Warning